src/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo
+  onRemoveTodo: (todoId: number) => void
+  onCheckedTodo: (todoId: number) => void
+  loading: boolean
+  handleImputTodo: (e: React.ChangeEvent<HTMLInputElement>) => void
+
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  onRemoveTodo,
+  onCheckedTodo,
+  loading,
+  handleImputTodo,
+}) => {
+  const [activeRenameTodo, setActiveRenameTodo] = useState<boolean>(false);
+
+  const handleTargetTodo = (
+    todoId: number,
+  ) => {
+    setActiveRenameTodo(() => todo.id === todoId);
+  };
+
+  return (
+    <>
+      <div
+        key={todo.id}
```

Comment:
You should add a key to the list item directly. No need to use the key attribute here
```
<TodoItem key={....} ....